quick-start/examples/n4.min.js

```diff
@@ -0,0 +1,15 @@
+import { Scene, Circle } from "https://unpkg.com/pencil.js/dist/pencil.esm.js";
+
+// Any CSS valid color is supported (named, hex, rgb, hsl ...)
+const scene = new Scene(undefined, {
+    fill: "#45a8ff",
+});
+
+const sun = new Circle(scene.center, 100, {
+    fill: "orange",
+});
+
+// The add function return the scene so we can chain calls
```

Comment:
```suggestion
// The "add" function returns the scene so we can chain calls
```
Adding the quotes might help to recognize `add` as a function name. You've done this below for `"draw" event`.